src/main/scala/com/thirdship/libunit/CompoundTSUnit.scala

```diff
@@ -165,17 +165,22 @@ class CompoundTSUnit(val numerator: List[TSUnit] = List.empty[TSUnit],
 		}
 	}
 
+	private def keySetEquals(unit: CompoundTSUnit): Boolean = {
+		numeratorDimensions.keySet.equals(unit.numeratorDimensions.keySet) &&
```

Comment:
Can you verify something for me? I think the CompoundTSUnit case in isConvertible can use equateUnitLists, rather than what it's using now. I just need another set of eyes, to make sure I'm not crazy. If it can, then we have a redundancy in our code that I can fix.